assets/scss/pages/contact-page.scss

```diff
@@ -0,0 +1,109 @@
+.c-contact-page__section {
```

Comment:
@mgbelegu please use `p-` prefix for pages. Also, I think it would be better to name the main block like `p-contact` and then `p-contant__title` and so on.